website/project/views/node.py

```diff
@@ -836,6 +840,7 @@ def _view_project(node, auth, primary=False,
         'addon_widgets': widgets,
         'addon_widget_js': js,
         'addon_widget_css': css,
+        'google_dataset_tags': json.dumps(JsonLdMetadataSerializer().serialize(pls_gather_item_metadata(node))),
```

Comment:
as implemented, `pls_gather_item_metadata` is a waste of time, the result is unused